packages/product/src/models/product-variant.ts

```diff
@@ -122,6 +122,14 @@ class ProductVariant {
   @Property({ columnType: "text", nullable: true })
   product_id!: string
 
+  @ManyToOne(() => Product, {
+    onDelete: "cascade",
+    index: "IDX_product_variant_product_id",
+    fieldName: "product_id",
+    nullable: true,
```

Comment:
q: variants won't exist without a product, should this be flipped to false?

Product module was our first mikroorm experiment, we might've made it nullable because we were still trying to figure out how this all works. 😄

Reply:
@riqwan same as above, I'll tackle this separately (I have a todo for myself already)